pkg/utils/json.go

```diff
@@ -28,11 +29,6 @@ func ValidateJSON(path string) ([]byte, error) {
 
 // MergeJsonMaps merges two maps of type map[string]interface{}
 func MergeJSONMaps(a, b map[string]interface{}) map[string]interface{} {
```

Comment:
do you believe it may be best to just remove the function, and use maps.Copy directly in pkg/ssh/ssh.go instead of MergeJSONmaps?

Reply:
I still feel like it makes sense to keep a separate func

Reply:
it makes code more readable imho